guidance/README.adoc

```diff
@@ -0,0 +1,31 @@
+= Guiding principles for Automation Good Practices
+
+As an overall guiding principle for designing good automation, the Zen of Python, by Tim Peters, serves as a good guidepost to follow. The principles defined in this are very applicable and can give guidance when the better practice is unclear.
```

Comment:
```suggestion
As an overall guiding principle for designing good automation, the Zen of Python, by Tim Peters, serves as a good guidepost to follow.
The principles defined in this are very applicable and can give guidance when the better practice is unclear.
```
We follow https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line